### PR TITLE
Updates default gum and peer connection constraints.

### PR DESCRIPTION
**Pull request by damencho**

### CODE REVIEW

**Comment by virtuacoplenny** [comment]
The old getConstraints still exists (for now and maybe for eternity). You might want to add the new constraints there as well to get browsers other than firefox, newer chrome, and new safari.

**Comment by damencho** [comment]
Thanks, will update and test there.

**Comment by damencho** [comment]
Old constraints updated.

**Comment by virtuacoplenny** [comment]
Looks good to me. I'll wait for @bbaldino to chime in too.

File: modules/RTC/RTCUtils.js [hunk 1]
```diff
@@ -275,7 +275,7 @@ function getConstraints(um, options) {
                 { echoCancellation: !disableAEC && !disableAP },
                 { googEchoCancellation: !disableAEC && !disableAP },
                 { googAutoGainControl: !disableAGC && !disableAP },
-                { googNoiseSupression: !disableNS && !disableAP },
+                { googNoiseSuppression: !disableNS && !disableAP },
```
Comment (bbaldino) [inline modules/RTC/RTCUtils.js]: nice catch

File: modules/RTC/RTCUtils.js [hunk 2]
```diff
@@ -1101,7 +1101,15 @@ class RTCUtils extends Listenable {
             || RTCBrowserType.isNWJS()
             || RTCBrowserType.isElectron()
             || RTCBrowserType.isReactNative()) {
-            this.pcConstraints = { optional: [] };
+            this.pcConstraints = { optional: [
```
Comment (bbaldino) [inline modules/RTC/RTCUtils.js]: maybe we should do an a/b test for these changes?

Comment (damencho) [reply modules/RTC/RTCUtils.js]: PSNR tests? And running with it for a week and checking overall callstats quality?

Comment (bbaldino) [reply modules/RTC/RTCUtils.js]: yeah i was thinking of doing something like what boris did for the video suspension change

**Pull request merged**